firebase-ai/api.txt

```diff
@@ -245,15 +247,15 @@ package com.google.firebase.ai.type {
   }
 
   public final class CountTokensResponse {
-    ctor public CountTokensResponse(int totalTokens, Integer? totalBillableCharacters = null, java.util.List<com.google.firebase.ai.type.ModalityTokenCount> promptTokensDetails = emptyList());
+    ctor public CountTokensResponse(int totalTokens, @Deprecated Integer? totalBillableCharacters = null, java.util.List<com.google.firebase.ai.type.ModalityTokenCount> promptTokensDetails = emptyList());
```

Comment:
@rlazo Is this from https://github.com/firebase/firebase-android-sdk/pull/7042?

Reply:
Oh, yes... absolutely. Will fix it in 

https://github.com/firebase/firebase-android-sdk/pull/7071

then you can sync